### PR TITLE
Make copy changes based on Vanessa feedback

### PR DESCRIPTION
**Pull request by nathancashmoredwp**

### CODE REVIEW

**Comment by nathancashmoredwp** [comment]
This is bullsh!t.  Can see the errors but cannot even create test db locally.  Read me needs some additions.

**Comment by gjvis** [comment]
Sorry :(

bin/test will run all tests

RAILS_ENV=test bin/rake db:setup

To prep test db. If that doesn't work, try 

bin/rake db:test:prepare

**Comment by nathancashmoredwp** [comment]
Sorry for being :angry: ... that person is now gone.  Long live :smiley: Nate.  Still need to add some styling for Jes.

File: app/views/scrapbooks/show.html.erb [hunk 1]
```diff
@@ -5,9 +5,14 @@
   </ol>
 <% end %>
 
-  <h2 class="heading-xlarge">Your work search activities</h2>
-</header>
-
+  <h2 class="heading-xlarge">Your saved roles</h2>
+<p>Now, you can now use this information to:</p>
```
Comment (gjvis) [inline app/views/scrapbooks/show.html.erb]: indentation

File: features/step_definitions/save_occupations_steps.rb [hunk 2]
```diff
@@ -12,7 +12,7 @@
 end
 
 Then(/^I should see a confirmation that the occupation is saved$/) do
-  confirmation_message = %("#{specific_search_result.fetch(:title)}" saved)
+  confirmation_message = %(#{specific_search_result.fetch(:title)} saved)
```
Comment (gjvis) [inline features/step_definitions/save_occupations_steps.rb]: May as well flip this back to using double quotes. I shouldn't have done something ~~clever~~ confusing in the first place

**Comment by gjvis** [comment]
:ship: :ship: :ship: :ship: :ship: :ship:

**Comment by gjvis** [comment]
:shipit:

**Pull request merged**